docs/features/placeholder.md

```diff
@@ -0,0 +1,71 @@
+---
+category: features
+---
+
+# Editor placeholder
+
+CKEditor 5 can display a configurable placeholder text when the content is empty. The placeholder helps users locate the editor in the application and prompts to input the content. It works similarly to the native DOM [`placeholder` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#The_placeholder_attribute) used by inputs.
+
+See the demo of the placeholder feature:
+
+{@snippet features/placeholder}
+
+## Configuring the placeholder
+
+There are two different ways of configuring the editor placeholder text:
+
+### Using the DOM attribute
+
+Set the `placeholder` attribute on an element passed to the `Editor.create()` method (for instance {@link module:editor-classic/classiceditor~ClassicEditor.create `ClassicEditor.create()`}) to configure the placeholder:
+
+```html
+<div id="editor" placeholder="Type the content here!">
```

Comment:
This does not look like a valid HTML. Could you verify this?

Reply:
![image](https://user-images.githubusercontent.com/1099479/52554531-6d8a8280-2de7-11e9-8422-70407a73731d.png)
You're right, we should use `data-placeholder` instead.

Reply:
https://github.com/ckeditor/ckeditor5/issues/1517